addons/point_of_sale/static/src/app/navbar/sale_details_button/sales_detail_report.xml

```diff
@@ -23,7 +23,7 @@
                         <div class="responsive-price">
                             <t t-esc="line['product_name'].substr(0,20)" />
                             <span class="pos-receipt-right-align">
-                                <t t-esc="Math.round(line.quantity * Math.pow(10, pos.dp['Product Unit of Measure'])) / Math.pow(10, pos.dp['Product Unit of Measure'])" />
+                                <t t-esc="Math.round(line.quantity * Math.pow(10, pos.models['decimal.precision'].find((dp) => dp.name === 'Product Unit of Measure').digits)) / Math.pow(10, pos.models['decimal.precision'].find((dp) => dp.name === 'Product Unit of Measure').digits)" />
```

Comment:
if this template is an owl component, put this calculation in the class (js).

Reply:
Its just a template